### PR TITLE
Add Fanvil X[3-6]U & X210

### PR DESCRIPTION
**Pull request by slemoal-oci**
Ajout des modèles fanvil
X3U, X4U, X5U, X6U, X210

### CODE REVIEW

**Review by bloom1 — CHANGES_REQUESTED** [review]
Hello @slemoal-oci!

Thank you for the PR :smile:

Two points:
- There are multiple `*.bak` files in the diff. Please make sure to remove these!
- If you want the plugin to be built by our system and upgraded by everyone, you must increment the plugin version number in the `plugin-info` file. 

Thank you!

File: plugins/wazo-fanvil/serie-x/pkgs/pkgs.db [hunk 1]
```diff
@@ -86,11 +121,21 @@ url: http://download.fanvil.com/Firmware/Release/X3S/20191210/x3s2.10.2.6887T201
 size: 3902427
 sha1sum: 9d16b8936ecf2b1ccd394aaf6e044faef7bb45f1
 
+[file_X3U-fw]
+url: https://download.fanvil.com/Firmware/Release/X3U/20211222/x3u-5912-RECOVERY-P0.18.23.1.23-2.4.12-1164T2021-11-10-01.18.44.z
+size: 11129914
+sha1sum: d18508f3c83d625e4e192dc08bd0d023
```
Comment (bloom1) [inline plugins/wazo-fanvil/serie-x/pkgs/pkgs.db:127]: The `sha1sum` are not valid for all the new firmwares
It is not the same as an `md5`

**Review by bloom1 — CHANGES_REQUESTED** [review]

**Comment by sopelj** [comment]
recheck

**Comment by fblackburn1** [comment]
@bloom1 any blockers on this PR?

**Comment by sopelj** [comment]
> @bloom1 any blockers on this PR?

Most of these phones (all but the X3U, are now supported in master) were added in another PR (#157) at the same time as the pros (PR #156). Because the way the config files were loaded needed to be changed in order to support firmware upgrades on both the "pro" and "non pro" models, since they use the same config names, but require different firmware.
 
The newer x-series phones should now extend `new-base.tpl`. This PR could possible be updated to just add the X3U phone.

**Comment by bloom1** [comment]
No reply from PR author, closing it.

**Pull request closed without merging**